cvent-technology-radar.csv

```diff
@@ -94,42 +95,43 @@ Jest,Adopt,Tools,FALSE,"<p>Jest is the unit testing tool we use for typescript/j
 Oracle,Hold,Platforms,FALSE,"<p>Oracle is another database leveraged heavily by Passkey, Conference and Business Transient product lines. However, as we continue to evolve we want to limit our investment in expanding the use of this database and leverage AWS Aurora</p>"
 Phrase App,Adopt,languages-and-frameworks,FALSE,"<p>A localization framework we're leveraging to localize our products</p>"
 pitest,Adopt,Tools,FALSE,"<p>A tool that helps you identify good tests from bad tests. This helps you avoid teams building tests that simply get code coverage without real test assertions</p>"
-Playwright,Adopt,Tools,TRUE,"<p>Playwright is a test automation framework we're using for end to end testing of our web applications</p>"
+Playwright,Adopt,Tools,FALSE,"<p>Playwright is a test automation framework we're using for end to end testing of our web applications</p>"
 pnpm,Adopt,Tools,FALSE,"<p>The build tool used by node projects and mono-repositories at Cvent.</p>"
 Postman,Hold,Tools,FALSE,"<p>We leverage postman as a tool for developing/testing APIs adhoc. Recent pricing changes & security concerns has pushed us away from using this going forward.</p>"
 Prisma,Hold,languages-and-frameworks,FALSE,"<p>Prisma is an interesting ORM that works well with graphql + typescript but has several short comings when it comes to how it handles transactions and joins. Prisma also uses a sidecar process to execute queries that severely hinders observability tools like Datadog. The side car process complicates connection pooling and IAM authentication and the sql generated is un friendly to RDS Proxy making it an unsuitable tool for serverless deployments. The approach we've taken to handle IAM results in memory leaks. For this reason we’re putting this technology on Hold and looking for alternatives.</p>"
 Python,Hold,languages-and-frameworks,FALSE,"<p>Python is primarily used for scripting, data science and lambda functions where appropriate but the preference is to use TypeScript where possible</p>"
 Quarkus,Hold,languages-and-frameworks,FALSE,"<p>We've chosen spring boot as go forward in 2024.</p>"
 RabbitMQ,Hold,Platforms,FALSE,"<p>We prefer that teams now use modern cloud native serverless messaging systems around our event driven architecture.</p>"
 RAML,Hold,Tools,FALSE,"<p>RAML is an api definition format for REST services. RAML is comparable to OpenAPI. We've consolidated our api definition language on Open API as it seems to have become the industry standard</p>"
-Rancher Desktop,Adopt,Tools,TRUE,"<p>We use rancher desktop to run containers on our local machines.</p>"
+Rancher Desktop,Adopt,Tools,FALSE,"<p>We use rancher desktop to run containers on our local machines.</p>"
 React Native,Adopt,languages-and-frameworks,FALSE,"<p>Our primary UI framework for mobile</p>"
 react-testing-library,Adopt,Tools,FALSE,"<p>The defacto standard tool for React unit testing</p>"
 ReactJS,Adopt,languages-and-frameworks,FALSE,"<p>Our primary UI framework for the web</p>"

```

Comment:
Add Roo under trial?

Reply:
Yes, but I wasn't sure if we wanted to add VSCode extensions here. This could explode the number of things we maintain in this radar. We don't even currently have VSCode or IntelliJ in the radar.

Should we? Where do we draw the line on what's useful for teams?

I did call out Roo here on the wiki https://wiki.cvent.com/pages/viewpage.action?pageId=856134918#ArtificialIntelligence(AI)-DeveloperTools

Reply:
Makes sense to leave it out.